src/logic/operate.js

```diff
@@ -0,0 +1,21 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result = 0;
+  const numberOneBig = Big(numberOne);
+  const numberTwoBig = Big(numberTwo);
+  if (operation === '-') {
+    result = numberOneBig - numberTwoBig;
+  } else if (operation === '+') {
+    result = numberOneBig + numberTwoBig;
+  } else if (operation === 'X') {
+    result = numberOneBig * numberTwoBig;
+  } else if (operation === '/') {
+    result = numberOneBig / numberTwoBig;
+  } else if (operation === '%') {
+    result = (numberOneBig / numberTwoBig) * 100;
```

Comment:
- Use `numberOneBig.mod(numberTwoBig) * 100`. The `mod` function will handle for you the rounding of the result and it comes with the `Big.js` package.